dirty.rb

```diff
@@ -0,0 +1,10 @@
+begin
+  1
+rescue
+end
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
__[development][rubocop]__
* Line is too long. [147/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)